.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,10 +1,30 @@
 ---
-name: Bug report
-about: Create a report to help us improve
-title: ''
+name: Bug Report
+about: Create a report to help us improve the EditorConfig.
+title: "[BUG] "
 labels: bug
 assignees: ''
 
 ---
 
-<!-- This issue tracker is only for generic bugs, such as those regarding the file format, or an issue that affects multiple editor plugins or core libraries. If this is a bug regarding a specific editor plugin or core library, please report to their respective issue tracker. -->
+<!--This issue tracker is only for generic bugs, such as those regarding the file format, or an issue that affects multiple editor plugins or core libraries. If this is a bug regarding a specific editor plugin or core library, please report to their respective issue tracker.-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Type '...'
+2. Open '...'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Device/Browser information:**
```

Comment:
Let's remove this section, because this tracker is for universal issues.

Reply:
Also forgot it was for browsers, not editors :)